R/save_arcadia_plot.R

```diff
@@ -0,0 +1,62 @@
+#' Save Arcadia Plot
+#'
+#' @param plot ggplot object to be saved
+#' @param filepath The path where the plot will be saved
+#' @param panel_size The size of the panel, one of "full_wide", "float_wide", "half_square", "full_square", "float_square"
+#' @param transparent_bg Logical, whether to set the plot background to be transparent
+#'
+#' @return None
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' p <- ggplot(mtcars, aes(x = hp, y = mpg)) + geom_point()
+#' save_arcadia_plot(p, "plot.pdf", "full_wide")
+#' }
+save_arcadia_plot <- function(filepath, plot, panel_size = c("full_wide", "float_wide", "half_square", "full_square", "float_square")) {
+  # Ensure the panel_size argument is valid
+  panel_size <- tryCatch(match.arg(panel_size), error = function(e) NA)
+
+  if (is.na(panel_size)) {
+    stop("Invalid panel_size. Please choose from: 'full_wide', 'float_wide', 'half_square', 'full_square', 'float_square'.")
```

Comment:
I think choosing a panel size should be optional. Not all data will fit neatly into the pre-determined panel sizes. In `arcadia_pycolor`, we separate the panel size designation from the save function; users can use the panel sizes if they want to, but are free to choose other sizes if they want. The panel sizes are just to make things simpler for those who want to use the Creative Cloud templates.

Reply:
Don't have to resolve this by tomorrow though; feel free to make an issue.

Reply:
Previously I had `landscape` and `portrait` as options, so I could just add those back in where those were just the default 8x11 or 11x8

Reply:
That seems like a reasonable solution for now!